static/js/publisher/pages/Releases/components/releasesTable/cellViews.js

```diff
@@ -68,13 +68,15 @@ EmptyInfo.propTypes = {
 };
 
 const ProgressiveTooltip = ({ revision, previousRevision }) => {
+  const { progressive } = revision?.releases?.[0];
+
   const previousRevisionData = (
     <>
       <strong>{previousRevision?.revision || "Unknown"}</strong>
       <br />
       <strong>
-        {Math.round(100 - revision?.progressive?.["current-percentage"]) || 0}%
-        → {Math.round(100 - revision?.progressive?.percentage)}%
+        {Math.round(100 - progressive?.["current-percentage"]) || 0}% →{" "}
```

Comment:
It's not wrong, but is it necessary to have the condition at revision and progressive?
Would this component be ever called without a revision that is progressive?

I feel like it should be safe to do `revision.progressive.` without `?`, and if we need a failsafe it could be eariler, as the whole tooltip would not make sense if there is no progressive revision to show?